app/real-estate-app/src/index.html

```diff
@@ -10,6 +10,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   </head>
+  <script src="https://maps.googleapis.com/maps/api/js?libraries=places&key=<API_KEY>>"></script>
```

Comment:
Shouldn't this be under the body tag?

Reply:
Based on https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup, I think we should use async and place it in the <head> tag